code/modules/antagonists/heretic/knowledge/ash_lore.dm

```diff
@@ -72,6 +72,8 @@
 
 /datum/heretic_knowledge/ashen_grasp/proc/on_mansus_grasp(mob/living/source, mob/living/target)
 	SIGNAL_HANDLER
+	if(!ismob(target))
```

Comment:
```suggestion
	if(!istype(target))
```
if you are assuming that its a living type in the proc body, then we might as well verify this assumption since if they somehow click on a non-living mob it will also fail